src/types/types.ts

```diff
@@ -242,6 +244,7 @@ export type VideoFormat =
   | "ogv"
   | "webm"
 
+export type AnimatedFormat = "auto" | "gif" | "webp" | "png";
```

Comment:
the name `AnimatedFormat` already existss in the codebase, maybe `AnimatedFormatType`?

Reply:
done